### ISSUE TITLE
Httpie can't resolve "localhost" in an Akka HTTP RESTful service

### ISSUE THREAD

**Issue by kun-song** [jakubroztocil/httpie#619]
I have a demo service like this:

```scala
localhost:8080/item/{id}
```

But when I test it with Httpie, I get the following message:

```
 $ http localhost:8080/item/3

http: error: ConnectionError: ('Connection aborted.', RemoteDisconnected('Remote end closed connection without response',)) while doing GET request to URL: http://localhost:8080/item/3
```

I tried 127.0.0.1 instead of localhost, and it works as expected:

```
 $ http 127.0.0.1:8080/item/3       
                                                                                                                                                                                                                            
HTTP/1.1 404 Not Found
Content-Length: 83
Content-Type: text/plain; charset=UTF-8
Date: Sat, 07 Oct 2017 08:55:38 GMT
Server: akka-http/10.0.9

The requested resource could not be found but may be available again in the future.
```

**Comment by nudemeth** [comment]
Hi

I'm not sure whether it is related to akka-http itself or not but I came here because I kind of faced this problem and found this page from Google. Although my problem is about Apache Bench (ab). I can use ab command to perform load test on 127.0.0.1 but was unable on localhost.

From this [link](https://stackoverflow.com/questions/36455497/listening-to-akka-http-ports-remotely), after I changed my binding point in akka-http from **localhost** to **0.0.0.0**, now 127.0.01, localhost and [::1] are all working fine.

Hope it helps.
